arcaflow_plugin_horreum_client/horreum_client_schema.py

```diff
@@ -31,31 +31,14 @@ class InputParams:
             )
         ),
     ]
-    horreum_keycloak_url: typing.Annotated[
+    api_key_pattern = re.compile(
+        r"HUSR_[A-Z0-9]{8}_[A-Z0-9]{4}_[A-Z0-9]{4}_[A-Z0-9]{4}_[A-Z0-9]{12}"
```

Comment:
Huh ... so it's a standard UUID with a prefix, except with underscores instead of hyphens? Odd.

Reply:
the prefix and overall format allow fast identification that a string is an Horreum API key and not just some UUID. (this is a common procedure in order to be able to grep for leaked keys)

the underscores also allow for selecting the key quickly with a double click :wink: